paladin/README.md

```diff
@@ -0,0 +1,238 @@
+
+```typescript
+import { Drash } from "https://deno.land/x/drash@{version}/mod.ts";
+
+// Import the Paladin middleware function
+import { Paladin } from "https://deno.land/x/drash-middleware@{version}/paladin/mod.ts";
+
+// Instantiate paladin
+const paladin = Paladin();
+
+// Create your server and plug in paladin to the middleware config
+const server = new Drash.Http.Server({
+  resources: [
+    HomeResource,
+  ],
+  middleware: {
+    after_request: [
+      paladin
+    ]
+  }
+});
+
+server.run({
+  hostname: "localhost",
+  port: 1447,
+});
+
+console.log(`Server running at ${server.hostname}:${server.port}`);
+```
+
+## Configuration
+
+Paladin has a list of default headers it will set, when `Paladin` is called with no arguments, but you can override these.
+
+### `X-XSS-Protection`
+
+This header does not protect against XSS attacks, but only protects against *one type* of XSS attack.
+
+See here for more information: https://helmetjs.github.io/docs/xss-filter/
+
+- [x] Enabled by default?
+
+```typescript
+const paladin = Paladin() // Set by default
+const paladin = Paladin({
+  "X-XSS-Protection": false // Disable it
+})
+const paladin = Paladin({
+  "X-XSS-Protection": true // Explicitly enable it
+})
+```
+
+### `Referrer-Policy`
+
+The Referrer-Policy header is usually set by browsers to tell a server where it's coming from.
+
+See here for more information: https://helmetjs.github.io/docs/referrer-policy/
+
+- [ ] Enabled by default?
+
+```typescript
+const paladin = Paladin() // Will not set the header
+const paladin = Paladin({
+  "Referrer-Policy": "origin" // Enable it, and set it to "origin"
+})
+```
+
+See [here](https://www.w3.org/TR/referrer-policy/#referrer-policies) for all possible policies.
+
+### `X-Content-Type-Options`
+
+The Referrer-Policy header is used to tell browsers not to sniff the mime type. Browsers will trust what the server says and block the resource if it’s wrong.
+
+See here for more information: https://helmetjs.github.io/docs/dont-sniff-mimetype/
+
+- [x] Enabled by default?
+
+```typescript
+const paladin = Paladin() // Set by default
+const paladin = Paladin({
+  "X-Content-Type-Options": false // Disable it
+})
+const paladin = Paladin({
+  "X-Content-Type-Options": true // Explicitly enable it
+})
+```
+
+### `Strict-Transport-Security`
+
+The Strict-Transport-Security (HSTS) header is used to tell browsers to stick with HTTPS and never visit the HTTP version.
+
+See here for more information: https://helmetjs.github.io/docs/hsts/
+
+- [x] Enabled by default?
+
+```typescript
+const paladin = Paladin() // Set by default to 60 days and include sub domains
+const paladin = Paladin({
+  hsts: {
+    maxAge: false // Disable the header altogether
+  }
+})
+const paladin = Paladin({
+  hsts: {
+    maxAge: 5184000 // Enable HSTS and set a max age
+  }
+})
+const paladin = Paladin({
+  hsts: {
+    includeSubDomains: false // Enable default max age but disable the inclusion of sub domains
```

Comment:
change to `include_subdomain`

Reply:
https://github.com/drashland/deno-drash-middleware/pull/25/commits/b8b3bba86166d57696b710bdb3513861ded928cd